### ISSUE TITLE
UI: Create help doc mapping via config.json

### ISSUE THREAD

**Issue by rhtyd** [apache/cloudstack#4731]
The helpdoc base URL is configurable via the config.json but not the query/sub-path which is configured in the config/ js files. A solution to make that customisable without makeing src code changes is to simply provide a translation logic/option in the config.json where users can simply specify the old path and new path as follows and the router handler/logic updated these overide mappings against what is described in source code:

```
"helpDocMappings": {
"virtual_machines.html": "some/new/path.html"
}""
```

##### ISSUE TYPE

 * Enhancement Request

##### COMPONENT NAME

~~~
UI
~~~

**Comment by shwstppr** [comment]
Addressed this in #4745 
For consistency key used for mappings `docHelpMappings` cc @rhtyd

**Comment by rhtyd** [comment]
Addressed this in #4745
